podvm/files/etc/systemd/system/attester.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Attester Service
+After=network.target
+
+[Service]
+Type=simple
+ExecStartPre=mkdir -p /run/confidential-containers
+ExecStart=/usr/local/bin/attester -s /run/confidential-containers/attester.sock
```

Comment:
Can we containerize this too? How we invoke it is yet to be decided. Either kata-agent invokes it, or for now even systemd can invoke it using runc.

Reply:
Sure, should be no problem. But it probably makes sense to align on an approach in [this discussion](https://github.com/confidential-containers/cloud-api-adaptor/issues/1220), right?